backend/Dockerfile

```diff
@@ -4,16 +4,20 @@ FROM golang:1.21-bookworm
 SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 
 # Install Dependencies
-RUN apt-get update && apt-get install -y --allow-downgrades \
-    inotify-tools=3.22.6.0-4             \
-    openssl=3.0.11-1~deb12u2
-
+#RUN apt-get update && apt-get install -y --allow-downgrades \
+#    inotify-tools=3.22.6.0-4             
+    
```

Comment:
* We still need inotify-tools for the dev version of this repo. This is used to watch for changes within the backend container so that the Golang binary can be re-written, so that developers have the experience of a dev mode container.

Reply:
dev version is not necessary. how can you debug a go program in docker ? by logs ?